### DISCUSSION TITLE
Is there a way to create nodes without supplying coordinates?

### DISCUSSION THREAD

**Discussion by wennerma** [xyflow/xyflow#1091] [Q&A]
When looking through the example I noticed nodes are usually provided coordinates.  Can this be used without proving that information?

**Comment by moklick** [comment]
Nope. That is not possible. What would you expect from a node without coordinates? should it be invisible?

**Reply by mniyad** [reply]
How about a simple 'default' layout. Perhaps a breadthfirst tree.
